src/main.rs

```diff
@@ -115,11 +115,19 @@ struct Exec {
     code: PathBuf,
 }
 
+/// Get a report from a keep
+#[derive(StructOpt)]
+struct Report {
+    /// The payload to run inside the keep
+    code: PathBuf,
+}
```

Comment:
For SGX we've got a `Report` type [here](https://github.com/enarx/sgx/blob/master/src/attestation_types/report.rs#L59) already that conforms to the SGX spec. It seems better to use this directly, unless there is some reason we can't.

Reply:
This is a command-line arg struct. `StructOpt` expands this such that one can do `enarx-keepldr report --code "potato.wasm"`.

Reply:
Ahhh, makes sense. Very good.